<?xml version="1.0"?>
<launch>
    <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
        <param name="destination_frame" value="car/base_link"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/scan_multi"/>
        <param name="laserscan_topics" value ="/scan"/>
	</node>
    <node pkg="nodelet" type="nodelet" name="pcl_manager" args="manager"
    output="screen" />

    <node pkg="nodelet" type="nodelet" name="outlier_removal" args="load
    pcl/StatisticalOutlierRemoval pcl_manager" output="screen">
        <remap from="~input" to="/merged_cloud"/>
        <remap from="~output" to="/merged_cloud_filtered"/>
        <rosparam>
            mean_k: 1
            stddev: 5.0
            negative: False
        </rosparam>
    </node>
</launch>
